api/bucket.go

```diff
@@ -192,12 +182,13 @@ func (h *BucketHandler) getDirArchive(ctx *gin.Context, dir string) {
 
 	fromTar := bufio.NewReader(&tarOutput)
 
-	ctx.Writer.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", path.Base(dir)+".tar.gz"))
```

Comment:
What is the Content-Disposition header for?

Reply:
It sets the downloaded filename; if ommited, the result file will not end .tar.gz